improver_tests/blending/test_utilities.py

```diff
@@ -158,116 +298,249 @@ def test_update_blended_metadata(model_cube, forecast_period):
         assert collapsed_cube.coord("forecast_period").points[0] == 2 * 3600
 
 
-@pytest.mark.parametrize(
-    "indices, expected",
-    (
-        ([0, 1], "uk_det:20171110T0100Z:\nuk_ens:20171110T0100Z:"),
-        ([0, 0], "uk_det:20171110T0100Z:"),
-    ),
-)
-def test_set_record_run_attr_basic(model_cube, indices, expected):
+def test_store_record_run_attr_basic(model_cube):
     """Test use case where the record_run_attr is constructed from other
-    information on the cubes. There are two tests here:
-
-      - cubes with unique attributes that are combined
-      - cubes with identical attributes attributes that are combined such that
-        only a single entry is returned."""
+    information on the cubes. The resulting cubes have an additional
+    auxiliary RECORD_COORD that stores the cycle and model information."""
 
     record_run_attr = "mosg__model_run"
     model_id_attr = "mosg__model_configuration"
-    cubes = [model_cube[index] for index in indices]
+    cubes = cubes = [model_cube[0], model_cube[1]]
```

Comment:
Should this be this?
```suggestion
    cubes = [model_cube[0], model_cube[1]]
```

Reply:
I really wanted to make sure that cubes was set. Changed.